ImageNet/inference/Alipay_resnet101_npu_tensorflow.json

```
@@ -0,0 +1,22 @@
+{
+    "version": "v1.0",
+    "author": "Ai CognitiveComputing team in Alipay Group",
```

This might be a little cleaner:

```suggestion
    "author": "AI Cognitive Computing team in Alipay Group",
```